app/assets/stylesheets/custom.scss

```diff
@@ -320,4 +320,51 @@ span.image {
 .users .follow { 
     padding: 0;
 }
-        
\ No newline at end of file
+
+img {
+    border-style: none;
+}
+
+.SigninButton {
+    --height: 44px;
+    background-color: #fff;
```

Comment:
Specify a color code at the beginning of the file, for example: `$color-white: #fff;`